web/src/people.js

```diff
@@ -890,6 +890,19 @@ export function get_realm_users() {
     return [...active_user_dict.values()];
 }
 
+export function get_realm_humans() {
```

Comment:
Let's call this `get_realm_active_human_users` -- I think that better fits our naming conventions and is more clear about the detail that we're only including active users.  `get_realm_active_human_user_ids` -- if you want to do that in a followup commit, it'd be appreciated.

Reply:
Sure, I have renamed the functions as you suggested.